docs/topics/wasm/wasm-js-interop.md

```diff
@@ -443,11 +431,29 @@ fun main() {
 }
 ```
 
-With the `-Xwasm-attach-js-exception` compiler option enabled, the `JsException` type provides specific details from the JavaScript error.
-Without enabling this compiler option, `JsException` includes only a generic message stating that an exception was thrown while running JavaScript code.
+This exception handling works automatically in modern browsers
+that support the [`WebAssembly.JSTag`](https://webassembly.github.io/exception-handling/js-api/#dom-webassembly-jstag)
+feature:
 
-If you try to use a JavaScript `try-catch` expression to catch Kotlin/Wasm exceptions, it looks like a
-generic `WebAssembly.Exception` without directly accessible messages and data.
+* Chrome 115+
+* Firefox 129+
+* Safari 18.4+
+
+### Exception handling in older browsers {initial-collapse-state="collapsed" collapsible="true"}
+
+To use exception handling in older browsers, you have to configure the `JsException`
+type to include the original error message and stack trace from JavaScript.
+To do so, add the following compiler option to your `build.gradle.kts` file:
+
+```kotlin
```

Comment:
It will not help old browsers, but it's a way to get more or less the same in old toolchain for modern browsers.